modules/swiss/src/main/SwissJson.scala

```diff
@@ -48,12 +49,28 @@ final class SwissJson(
       socketVersion: Option[SocketVersion] = None,
       playerInfo: Option[SwissPlayer.ViewExt] = None
   )(implicit lang: Lang): Fu[JsObject] = {
+    val playerJsonFuture: Fu[JsObject] = playerInfo match {
```

Comment:
Can you refactor this into a function to keep inline with the logic on lines 68-74

Reply:
also I think you are calling swissApi.playerInfo when you don't need to, you already have this result from info?